packages/framework/src/Model/Order/Order.php

```diff
@@ -1010,7 +1022,7 @@ public function edit(
                 $orderItemData->priceWithoutVat = $orderItemPriceCalculation->calculatePriceWithoutVat($orderItemData);
                 $orderItem->edit($orderItemData);
             } else {
-                $this->removeItem($orderItem);
+                $this->removeProductTypeItem($orderItem);
```

Comment:
cant find it in BC break list so OK then.

Reply:
I don't think that this is BC break